### DISCUSSION TITLE
Update translation content

### DISCUSSION THREAD

**Discussion by soulteary** [outline/outline#2305] [General]
Hi, project maintainer.

I have updated some translation content in the translation portal. Will anyone review it?

At present, the Chinese terms have been completely translated. If the review is passed, can these translations be used in the next version?

Thanks.

**Comment by tommoor** [comment]
Hi, yes – the changes are automatically PR'd and will be included in the next version:
https://github.com/outline/outline/pull/2304/files

Thank you

**Reply by soulteary** [reply]
Thanks for the reply. 

During the in-depth use of the software, I found some inaccurate words and entries that were not covered by i18N. 

May I ask if there is a user manual for you to refer to if you need to make adjustments or corrections?

Thanks.

**Reply by tommoor** [reply]
Please see: https://github.com/outline/outline/blob/main/TRANSLATION.md
